docs/content/release_notes/changelog_1.8.0.md

```diff
@@ -1,6 +1,6 @@
 # 1.8.0 (December 2021)
 
-IBM® block storage CSI driver 1.8.0 added new support and enhancements:
+IBM® block storage CSI driver 1.8.0 adds new support and enhancements:
```

Comment:
why this change?

Reply:
because we do this between every release :)

Reply:
then how come below 1.8.0 we still have `added`? what are the rules?

Reply:
we've been through this before :)
the new one is active / present.
the older content is past tense.

So in 1.8.0 it would have been "adds" and we would have changed 1.7.0 to "added" (for example) i do this for all RN content across all of the products.

Reply:
I'm not sure I understand - in your 1.8.0 example, when (and why) do you change "added" to "adds" in relation to 1.8.0 GA?

Reply:
thanks for reverting this change!